customization/XFCE.md

```diff
@@ -39,7 +35,6 @@ Things needs/recommended to be done:
 -   create own favorites menu (currently standard XFCE menu isn't modified to use per-VM subsections, which makes it very inconvenient):
     1.  create \~/.config/menus/favorites.menu, example content:
 
-        ```
         <!DOCTYPE Menu PUBLIC "-//freedesktop//DTD Menu 1.0//EN"
```

Comment:
[Right](https://github.com/QubesOS/qubesos.github.io/blob/12dc47618b0bc4e20bf52dc31ff08223a3693486/_config.yml#L23), thanks for the hint.